tools/topology/topology2/platform/intel/hdmi-in-ssp-config.conf

```diff
@@ -0,0 +1,56 @@
+#SSP DAI configuration to receive samples via I2S(in slave mode)
+#from HDMI input converter chip.
+Object.Dai.SSP [
+	{
+		id				$HDMI_IN_1_ID
+		dai_index		$HDMI_IN_1_INDEX
+		direction		"capture"
+		name			$HDMI_IN_1_NAME
+		default_hw_conf_id	0
+		sample_bits		32
+		io_clk			$HDMI_IN_MCLK
+		Object.Base.hw_config.1 {
+			id		0
+			name		"HDMI-In-1"
+			mclk_freq	$HDMI_IN_MCLK
+			bclk_freq	3072000
+			tdm_slot_width	32
+			format		"I2S"
+			bclk		"codec_provider"
+			fsync		"codec_provider"
+			fsync_freq	48000
```

Comment:
if bclk and fsync are provided by the codec, do we actually need any values for bclk_freq and fsync_freq, since they would be *nominal* values and not the actual ones?

Reply:
As my understanding, the setting "codec_provider" will ignore all other clk params and expect codec will provide the clk. These values are just reference that what would be the expected. also my understanding is that , HDMI-in use case similar as BT offload where we configure I2S in slave mode.  There still we maintain the clk values in dai config. 

io_clk			$BT_MCLK
		Object.Base.hw_config.1 {
			id		0
			name		"BT-SCO-WB"
			mclk_freq	$BT_MCLK
			bclk_freq	256000
			bclk_invert     true
			tdm_slot_width	16
			format		"DSP_A"
			bclk		"codec_provider"
			fsync		"codec_provider"
			fsync_freq	16000

So I'm not sure that whether we can ignore these params in config and following the similar approach. 
please correct me if my understanding is not correct.